mapswipe_workers/mapswipe_workers/generate_stats/overall_stats.py

```diff
@@ -65,6 +66,11 @@ def get_project_static_info(filename: str) -> pd.DataFrame:
                 ,regexp_replace(look_for, E'[\\n\\r]+', ' ', 'g' ) as look_for
                 ,project_type
                 -- add an array of the tile server names
+                ,CASE
+                  WHEN project_type_specifics-> 'answerLabels' IS NOT NULL THEN
+                  ARRAY(SELECT json_array_elements(project_type_specifics->'answerLabels')->>'value')
+                  ELSE ARRAY(0,1,2,3)  -- mapswipe app default
+                END as answer_label_values
```

Comment:
this should be simplified and better explained

Reply:
Added tests [here 6505eed](https://github.com/mapswipe/python-mapswipe-workers/pull/794/commits/6505eedc0ea8856911d67dc8b351a11eaae3aa88)